e2e/framework/framework.go

```diff
@@ -158,6 +159,9 @@ func InitializeCluster() {
 	if mode == wizardMode {
 		Cluster, application, err = infra.NewWizard(config, provisioner, installerNode)
 		TestContext.Application.Locator = application
+		if err != nil {
+			testState.EntryURL = strings.Replace(Cluster.OpsCenterURL(), "61009", "32009", -1)
```

Comment:
In wizard mode we doesn't have defined ops_url in state. For test, like update, which use state, I need OpsCenter URL.